--- a/internal/graalvm_bindist.bzl
+++ b/internal/graalvm_bindist.bzl
@@ -80,10 +80,10 @@ def _get_platform(ctx, newdist):
     if not newdist or not versions.is_at_least("5", versions.get()):
         return _get_platform_legacy(ctx, not newdist)
     elif ctx.os.name == "linux":
-        return ("linux-%s" % (arch_labels[ctx.os.arch] or ctx.os.arch), "linux", "tar.gz")
+        return ("linux-%s" % (arch_labels.get(ctx.os.arch, ctx.os.arch) or ctx.os.arch), "linux", "tar.gz")
     elif ctx.os.name == "mac os x":
-        return ("macos-%s" % (arch_labels[ctx.os.arch] or ctx.os.arch), "macos", "tar.gz")
+        return ("macos-%s" % (arch_labels.get(ctx.os.arch, ctx.os.arch) or ctx.os.arch), "macos", "tar.gz")
     elif "windows" in ctx.os.name:
-        return ("windows-%s" % (arch_labels[ctx.os.arch] or ctx.os.arch), "windows", "zip")
+        return ("windows-%s" % (arch_labels.get(ctx.os.arch, ctx.os.arch) or ctx.os.arch), "windows", "zip")
     else:
         fail("Unsupported operating system: " + ctx.os.name)
 
